python/lsst/jointcal/jointcal.py

```diff
@@ -277,6 +277,15 @@ class JointcalTaskConnections(pipeBase.PipelineTaskConnections,
         multiple=True
     )
 
+    def __init__(self, *, config=None):
+        super().__init__(config=config)
+        if not config.doAstrometry:
```

Comment:
Is it worth putting a comment or docstring about what is going on here? It was not immediately obvious to me, but adding this `__init__` seems like a good thing.